application/templates/service.yaml

```diff
@@ -16,6 +16,9 @@ metadata:
   name: {{ template "application.name" . }}
   namespace: {{ template "application.namespace" $ }}
 spec:
+{{- if .Values.service.type }}  
```

Comment:
I think this condition is not required. As we are providing default value in pipeline. if we dont provide it will still be ClusterIP. 
in short, I dont think anyone will explicitly remove this field from default value

Reply:
@aliartiza75 ^^

Reply:
@hanzala1234 I agree but it would be good just to have the ability to configure it.